### ISSUE TITLE
Empty errors on large array

### ISSUE THREAD

**Issue by MoonlightSentinel** [dlang-community/D-Scanner#786]
Reduced from dmds `ptrntab.d`:

```
struct OP {
    const(char)* str;   // opcode string
    ubyte usNumops;
    PTRNTAB ptb;
}

union PTRNTAB {
        void *ppt;
}

immutable OP[] optab = [
        { "__emit",     ITdata | OPdb,  { null } },
        { "_emit",      ITdata | OPdb,  { null } },
        { "aaa",        0,              { aptb0AAA.ptr } },
        { "aad",        0,              { aptb0AAD.ptr } },
        { "aam",        0,              { aptb0AAM.ptr } },
        { "aas",        0,              { aptb0AAS.ptr } },
        { "adc",        2,              { aptb2ADC.ptr } },
        { "add",        2,              { aptb2ADD.ptr } },
        { "addpd",      2,              { aptb2ADDPD.ptr } },
        { "addps",      2,              { aptb2ADDPS.ptr } },
        { "addsd",      2,              { aptb2ADDSD.ptr } },
        { "addss",      2,              { aptb2ADDSS.ptr } },
        { "addsubpd",   2,              { aptb2ADDSUBPD.ptr } },
        { "addsubps",   2,              { aptb2ADDSUBPS.ptr } },
        { "aesdec",     2,              { aptb2AESDEC.ptr } },
        { "aesdeclast", 2,              { aptb2AESDECLAST.ptr } },
        { "aesenc",     2,              { aptb2AESENC.ptr } },
        { "aesenclast", 2,              { aptb2AESENCLAST.ptr } },
        { "aesimc",     2,              { aptb2AESIMC.ptr } },
        { "aeskeygenassist", 3,         { aptb3AESKEYGENASSIST.ptr } },
        { "and",        2,              { aptb2AND.ptr } },
        { "andnpd",     2,              { aptb2ANDNPD.ptr } },
        { "andnps",     2,              { aptb2ANDNPS.ptr } },
        { "andpd",      2,              { aptb2ANDPD.ptr } },
        { "andps",      2,              { aptb2ANDPS.ptr } },
        { "arpl",       2,              { aptb2ARPL.ptr } },
        { "blendpd",    3,              { aptb3BLENDPD.ptr } },
        { "blendps",    3,              { aptb3BLENDPS.ptr } },
        { "blendvpd",   3,              { aptb3BLENDVPD.ptr } },
        { "blendvps",   3,              { aptb3BLENDVPS.ptr } },
        { "bound",      2,              { aptb2BOUND.ptr } },
        { "bsf",        2,              { aptb2BSF.ptr } },
        { "bsr",        2,              { aptb2BSR.ptr } },
        { "bswap",      1,              { aptb1BSWAP.ptr } },
        { "bt",         2,              { aptb2BT.ptr } },
        { "btc",        2,              { aptb2BTC.ptr } },
        { "btr",        2,              { aptb2BTR.ptr } },
        { "bts",        2,              { aptb2BTS.ptr } },
        { "call",       ITjump | 1,     { aptb1CALL.ptr } },
        { "cbw",        0,              { aptb0CBW.ptr } },
        { "cdq",        0,              { aptb0CDQ.ptr } },
        { "cdqe",       0,              { aptb0CDQE.ptr } },
        { "clc",        0,              { aptb0CLC.ptr } },
        { "cld",        0,              { aptb0CLD.ptr } },
        { "clflush",    1,              { aptb1CLFLUSH.ptr } },
        { "cli",        0,              { aptb0CLI.ptr } },
        { "clts",       0,              { aptb0CLTS.ptr } },
        { "cmc",        0,              { aptb0CMC.ptr } },
        { "cmova",      2,              { aptb2CMOVNBE.ptr } },
        { "cmovae",     2,              { aptb2CMOVNB.ptr } },
        { "cmovb",      2,              { aptb2CMOVB.ptr } },
        { "cmovbe",     2,              { aptb2CMOVBE.ptr } },
        { "cmovc",      2,              { aptb2CMOVB.ptr } },
        { "cmove",      2,              { aptb2CMOVZ.ptr } },
        { "cmovg",      2,              { aptb2CMOVNLE.ptr } },
        { "cmovge",     2,              { aptb2CMOVNL.ptr } },
        { "cmovl",      2,              { aptb2CMOVL.ptr } },
        { "cmovle",     2,              { aptb2CMOVLE.ptr } },
        { "cmovna",     2,              { aptb2CMOVBE.ptr } },
        { "cmovnae",    2,              { aptb2CMOVB.ptr } },
        { "cmovnb",     2,              { aptb2CMOVNB.ptr } },
        { "cmovnbe",    2,              { aptb2CMOVNBE.ptr } },
        { "cmovnc",     2,              { aptb2CMOVNB.ptr } },
        { "cmovne",     2,              { aptb2CMOVNZ.ptr } },
        { "cmovng",     2,              { aptb2CMOVLE.ptr } },
        { "cmovnge",    2,              { aptb2CMOVL.ptr } },
        { "cmovnl",     2,              { aptb2CMOVNL.ptr } },
        { "cmovnle",    2,              { aptb2CMOVNLE.ptr } },
        { "cmovno",     2,              { aptb2CMOVNO.ptr } },
        { "cmovnp",     2,              { aptb2CMOVNP.ptr } },
        { "cmovns",     2,              { aptb2CMOVNS.ptr } },
        { "cmovnz",     2,              { aptb2CMOVNZ.ptr } },
        { "cmovo",      2,              { aptb2CMOVO.ptr } },
        { "cmovp",      2,              { aptb2CMOVP.ptr } },
        { "cmovpe",     2,              { aptb2CMOVP.ptr } },
        { "cmovpo",     2,              { aptb2CMOVNP.ptr } },
        { "cmovs",      2,              { aptb2CMOVS.ptr } },
        { "cmovz",      2,              { aptb2CMOVZ.ptr } },
        { "cmp",        2,              { aptb2CMP.ptr } },
        { "cmppd",      3,              { aptb3CMPPD.ptr } },
        { "cmpps",      3,              { aptb3CMPPS.ptr } },
        { "cmps",       2,              { aptb2CMPS.ptr } },
        { "cmpsb",      0,              { aptb0CMPSB.ptr } },
        { "cmpsd",      ITopt|3,        { aptb3CMPSD.ptr } },
        { "cmpsq",      0,              { aptb0CMPSQ.ptr } },
        { "cmpss",      3,              { aptb3CMPSS.ptr } },
        { "cmpsw",      0,              { aptb0CMPSW.ptr } },
        { "cmpxchg",    2,              { aptb2CMPXCHG.ptr } },
        { "cmpxchg16b", 1,              { aptb1CMPXCH16B.ptr } },
        { "cmpxchg8b",  1,              { aptb1CMPXCH8B.ptr } },
        { "comisd",     2,              { aptb2COMISD.ptr } },
        { "comiss",     2,              { aptb2COMISS.ptr } },
        { "cpuid",      0,              { aptb0CPUID.ptr } },
        { "cqo",        0,              { aptb0CQO.ptr } },
        { "crc32",      2,              { aptb2CRC32.ptr } },
        { "cvtdq2pd",   2,              { aptb2CVTDQ2PD.ptr } },
        { "cvtdq2ps",   2,              { aptb2CVTDQ2PS.ptr } },
        { "cvttsd2si",  2,              { aptb2CVTTSD2SI.ptr } },
        { "cvttss2si",  2,              { aptb2CVTTSS2SI.ptr } },
        { "cwd",        0,              { aptb0CWD.ptr } },
        { "cwde",       0,              { aptb0CWDE.ptr } },
        { "da",         ITaddr | 4,     { null } },

];
```

Checking this using dmds `dsanner.ini` yields multiple empty error messages which disappear if any further line is omitted.

```
.\stylecheck.d(11:22)[error]:
```

**Comment by MoonlightSentinel** [comment]
Introduced in 19e9b9093a5fa33ace9ad9d3ac801a8090c18b2a

**Comment by MoonlightSentinel** [comment]
Fix for the empty error messages: #787.

These errors are caused by `libdparse`, see dlang-community/libdparse#372
